components/automate-ui/src/app/modules/team/team-management/team-management.component.html

```diff
@@ -6,31 +6,22 @@
           <chef-subheading>Chef Automate only displays local teams.</chef-subheading>
         </chef-page-header>
 
-      <app-create-v1-team-modal
-        [visible]="createV1TeamModalVisible"
-        [creating]="creatingTeam"
-        [conflictErrorEvent]="conflictErrorEvent"
-        [createForm]="createV1TeamForm"
-        (close)="closeCreateModal()"
-        (createClicked)="createV1Team()">
-      </app-create-v1-team-modal>
-
       <app-create-object-modal
         [visible]="createModalVisible"
         [creating]="creatingTeam"
         [conflictErrorEvent]="conflictErrorEvent"
         objectNoun="team"
         [createForm]="createTeamForm"
-        [showProjectsDropdown]="isIAMv2"
+        [showProjectsDropdown]="true"
```

Comment:
Now we could even eliminate this parameter completely if we modify the modal to just check for `objectNoun != 'project'`. That alleviates a bit of the code smell of having 9 parameters(!) to the component.

Reply:
It's still used in the project create modal.

Reply:
Exactly, please re-read my comment. 😁

Reply:
➕ to removing the `showProjectsDropdown` parameter